doc/api/typescript.md

```diff
@@ -68,6 +68,18 @@ By intentionally not supporting syntaxes that require JavaScript code
 generation, and by replacing inline types with whitespace, Node.js can run
 TypeScript code without the need for source maps.
 
+The suggested `tsconfig.json` settings for type stripping are:
+
+```json
+{
+    "compilerOptions": {
+        "allowImportingTsExtensions": true,
```

Comment:
```suggestion
        "module": "nodenext",
        "allowImportingTsExtensions": true,
```

Actually, this may be a good idea.

Reply:
See: https://www.typescriptlang.org/docs/handbook/modules/guides/choosing-compiler-options.html#im-compiling-and-running-the-outputs-in-nodejs (roughly)